src/vs/platform/environment/common/environmentService.ts

```diff
@@ -14,6 +14,8 @@ import { NativeParsedArgs } from 'vs/platform/environment/common/argv';
 import { ExtensionKind, IDebugParams, IExtensionHostDebugParams, INativeEnvironmentService } from 'vs/platform/environment/common/environment';
 import { IProductService } from 'vs/platform/product/common/productService';
 
+export const EXTENSION_IDENTIFIER_WITH_LOG_REGEX = /^([^.]+\..+):(.+)$/;
```

Comment:
Can extension names contain colons? I not, the regex could be written a bit more restrictive to avoid backtracking
e.g. [^.:]+\.[^:]+

Reply:
`:` is not a valid character in extension name. So we are good here.